include/inria_wbc/controllers/controller.hpp

```diff
@@ -28,6 +28,22 @@
 #include <inria_wbc/utils/utils.hpp>
 
 namespace inria_wbc {
+
+    namespace mask
+    {
+        using mask6 = Eigen::Array<double, 6, 1>;
+
+        const mask6 all = (mask6() << 1, 1, 1, 1, 1, 1).finished();
```

Comment:
We need the static (static const) here, otherwise we will have issues if we use the header twice. This is still not fully standard compliant but if it works with gcc and clang, I think we should not bother much...

Reply:
right.